tests/dev_config.toml

```diff
@@ -0,0 +1,14 @@
+[sqlalchemy]
```

Comment:
Why is this file necessary?

Reply:
It's used as the configuration for the quetz instance in the README script, see [here](https://github.com/mamba-org/quetz-client/blob/44a27e3cb8e9aa4672b62d4f375ce6579100ae96/.github/workflows/test-readme.sh#L7). The same file is also used by the live tests in #2.